cs-config/cs_config/functions.py

```diff
@@ -24,7 +24,7 @@
 TCDIR = os.path.dirname(TCPATH)
 with open(os.path.join(TCDIR, "policy_current_law.json"), "r") as f:
     pcl = json.loads(f.read())
-RES = convert_defaults(pcl)
+RES = pcl#convert_defaults(pcl)
```

Comment:
@hdoupe To have this run locally, I had to avoid the use the ``convert_defaults`` function here since Tax-Calc now uses paramtools (at least my local version does).  Do I need to keep this in because C/S is using an older TC version?

Reply:
Ah, I see. C/S is still on TC 2.9.0 which doesn't have ParamTools. If you'd like, you can run OG-USA with TC on the `master` branch. However, I think a major release is coming soon and it may be better to wait on that before swapping.

Reply:
@hdoupe I just wanted to confirm.  Let me test locally with the TC 2.9.   I'll merge PR after that completes successfully.